net/redistest/redistest.go

```diff
@@ -15,8 +15,9 @@ func NewTestRedis(t testing.TB) (address string, done func()) {
 }
 
 func NewTestRedisWithPassword(t testing.TB, password string) (address string, done func()) {
+	const dateFormat = "2006-01-02"
 
-	if fixDeadline, _ := time.Parse("2006-01-02", "2023-08-01"); time.Now().Before(fixDeadline) {
+	if fixDeadline, _ := time.Parse(dateFormat, "2023-08-08"); time.Now().Before(fixDeadline) {
```

Comment:
It seems to me we should postpone even more, something like 1st of September, maybe even later.

Reply:
Lets move the deadline for one more week and then explore alternative solutions, e.g. https://github.com/testcontainers/testcontainers-go/issues/1359#issuecomment-1657098312